script/test/cmd/cmd_test.go

```diff
@@ -0,0 +1,354 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus"
+	"github.com/joho/godotenv"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func fileNotExist(path string) bool {
+	return findExecutable(path) != nil
+}
+func findExecutable(file string) error {
+	d, err := os.Stat(file)
+	if err != nil {
+		return err
+	}
+	if m := d.Mode(); !m.IsDir() {
+		return nil
+	}
+	return os.ErrPermission
+}
+
+func binary(cmd string) string {
+	path, err := exec.LookPath(cmd)
+	if err != nil || fileNotExist(cmd) {
+		home := os.Getenv("HOME")
+		if home == "" {
+			fmt.Println("No $HOME environment variable found")
+		}
+	}
+	return path
+}
+
+// Kompose command
+func Kompose(command, provider, dockerComposeFile, result string) (cmd, output string) {
+	b, _ := ioutil.ReadFile(result)
+	file := strings.Split(dockerComposeFile, "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[1])
+		os.Exit(1)
+	}
+	cmd = string(out)
+	return cmd, output
+}
+
+// Kompose command for bundle
+func KomposeBundle(command, dockerComposeFile, result string) (cmd, output string) {
+	b, _ := ioutil.ReadFile(result)
+	file := strings.Split(dockerComposeFile, "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{"--bundle", dockerComposeFile, command, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[1])
+		os.Exit(1)
+	}
+	cmd = string(out)
+	return cmd, output
+}
+
+//GenerateArtifacts generates artifacts with -o command
+func GenerateArtifacts(command, provider, output_file, dockerComposeFile string) (outfile string) {
+	file := strings.Split(dockerComposeFile, "/")
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "-o", output_file, "-j"}
+	_, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Error Cmd", err, file[1])
+		os.Exit(1)
+	}
+	outfile = string(output_file)
+	return outfile
```

Comment:
we don't need to put in `return outfile` here, since the variable name is in the function definition above, we can just use `return`